bin/run_tests

```diff
@@ -19,11 +19,12 @@
 # Exit on non-zero exit code from subcommand
 set -e
 
+export PATH=$PATH:$(pwd)/bin
```

Comment:
This won't always work, you'll need to do something like `$(cd $(dirname $0) ; pwd)` instead.